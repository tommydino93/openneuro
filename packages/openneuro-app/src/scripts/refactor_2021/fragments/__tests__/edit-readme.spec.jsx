import React from 'react'
import { mount } from 'enzyme'
import EditReadme from '../edit-readme.jsx'

describe('EditReadme component', () => {
  it('renders with default props', () => {
    const wrapper = mount(<EditReadme />)
    expect(wrapper).toMatchSnapshot()
  })
})
