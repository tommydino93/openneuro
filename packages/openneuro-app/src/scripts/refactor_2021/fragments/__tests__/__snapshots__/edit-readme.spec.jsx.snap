// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditReadme component renders with default props 1`] = `<EditReadme />`;
